### DISCUSSION TITLE
Battery connection part

### DISCUSSION THREAD

**Discussion by ntukenmez3** [bitcraze/discussions#1533] [Q&A] [answered]
How can we fix this part? It's working however I think it needs better solution. Could you give us a suggestion to fix it in a better way?![image](https://github.com/user-attachments/assets/e12f4a83-f0d4-41d9-be56-8acea069c45e)

**Comment by tobbeanton** [comment, marked as answer]
I guess you are referring to the battery cable? Do you still have the old one? You can buy a spare, it is commonly referred to as [Molex 2.0mm 2P](https://hobbyking.com/en_us/molex-2-0-2-pin-cable-male-connector-with-200mm-x-24awg-silicone-wire-5pcs.html?___store=en_us) and it should have ~24AWG wires.

To remove the solder in the holes you need good heat transfer (flux&new solder) and possible increase the soldering temperature a bit to melt it though. Then use a solder sucker to get it out. "Banging" it out also works but should be done with some caution.

**Reply by ntukenmez3** [reply]
We're going to buy this connector. Thank you.
